core/src/components/input/test/basic/input.e2e.ts

```diff
@@ -185,6 +223,28 @@ configs({ modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
       const container = page.locator('#container');
       await expect(container).toHaveScreenshot(screenshot(`input-clear-button-focused`));
     });
+
+    test('should not have visual regressions when user dbclicks', async ({ page, browserName }) => {
+      if (browserName === 'firefox' || browserName === 'webkit') {
+        //TODO: Figure out why it's not working for webkit & firefox
+        test.skip();
+      }
+      await page.setContent(
+        `
+          <ion-input
+            label="Label"
+            label-placement="stacked"
+            clear-input="true"
+            value="Text"
+          ></ion-input>
+      `,
+        config
+      );
+      const input = page.locator('ion-input');
+      await input.dblclick();
```

Comment:
I think the reason that the native input should be used to double click on is because we verify that the mouse actually is on top of the input. If we use just `ion-input` then the Playwright mouse might be clicking on the right area to trigger a focus when using specific browsers.

```suggestion
      const input = page.locator('ion-input');
      const nativeInput = input.locator('input');
      await nativeInput.dblclick();
```